ci-operator/step-registry/openshift-qe/installer/bm/deploy/openshift-qe-installer-bm-deploy-commands.sh

```diff
@@ -15,6 +15,7 @@ LAB=$(cat ${CLUSTER_PROFILE_DIR}/lab)
 export LAB
 LAB_CLOUD=$(cat ${CLUSTER_PROFILE_DIR}/lab_cloud || cat ${SHARED_DIR}/lab_cloud)
 export LAB_CLOUD
+LAB_INTERFACE=$(cat ${CLUSTER_PROFILE_DIR}/lab_interface)
```

Comment:
Why are you adding this here?
We want jetlag to automatically discover the interfaces

Reply:
Also, I think you are not using it anywhere else in the code